PyDSS/valiate_settings.py

```diff
@@ -17,7 +17,6 @@
             'Export Data In Memory': {'type': bool, 'Options': [True, False]},
             'HDF Max Chunk Bytes': {'type': int, 'Options': range(16 * 1024, 1024 * 1024 + 1)},
             'Log Results': {'type': bool, 'Options': [True, False]},
-            'Result Container': {'type': str, 'Options': ['ResultContainer', 'ResultData']},
```

Comment:
There are many places in the code that still use this. Two locations are in PyDSS/dssInstance.py and those cause runtime exceptions. Those need to be removed.

Many example and test simulation.toml files also have this defined. Either they all need to be fixed or you could leave this setting defined (and ignored). It might be worthwhile to avoid breaking existing users' config files.

```
git grep -c "Result Container"
PyDSS/api/schema/PyDSS.v1.json:1
PyDSS/api/src/web/handler.py:1
PyDSS/api/src/web/parser.py:1
PyDSS/dssInstance.py:2
docs/source/tutorial.rst:1
examples/custom_contols/ProfileManager.toml:1
examples/custom_contols/simulation.toml:1
examples/custom_contols/vvar.toml:1
examples/dynamic_visualization/Scenarios/scenario_2/networkgraph.toml:1
examples/dynamic_visualization/Scenarios/scenario_2/simulation-run.toml:1
examples/dynamic_visualization/Scenarios/scenario_2/simulation.toml:1
examples/dynamic_visualization/networkgraph.toml:1
examples/dynamic_visualization/simulation.toml:1
examples/dynamics/simulation.toml:1
examples/external_interfaces/Python_example/smulation.toml:1
examples/external_interfaces/pydss_project/Scenarios/socket/simulation-run.toml:1
examples/external_interfaces/pydss_project/helics.toml:1
examples/external_interfaces/pydss_project/helics_itr.toml:1
examples/external_interfaces/pydss_project/simulation.toml:1
examples/harmonics/Scenarios/static/simulation-run.toml:1
examples/harmonics/simulation.toml:1
examples/harmonics/snapshot.toml:1
examples/monte_carlo/simulation.toml:1
examples/pv_validation/simulation.toml:1
```

Reply:
removed all instances